### PR TITLE
Use updated CV assets and show CV logo on splash screen.

### PR DESCRIPTION
**Pull request by manjotsidhu**
In this Pull Request, I have updated CircuitVerse logo in App Drawer and used the same logo to show in Splash Screen.

### Screenshots

Splash Screen
Before -> After
 

App Drawer
Before -> After

### CODE REVIEW

**Comment by Nitish145** [comment]
@nitin10s can u please have look here.

File: lib/ui/views/cv_landing_view.dart [hunk 1]
```diff
@@ -95,15 +94,9 @@ class _CVLandingViewState extends State<CVLandingView> {
...
-              'CircuitVerse',
-              textAlign: TextAlign.center,
-              style: Theme.of(context).textTheme.headline3.copyWith(
-                    fontFamily: 'Grinched',
-                    color: AppTheme.primaryColor,
-                  ),
-            ),
+            padding: const EdgeInsets.symmetric(vertical: 32, horizontal: 10),
+            child: Image.asset('assets/images/landing/cv_full_logo.png',
+                width: 90),
```
Comment (Nitish145) [inline lib/ui/views/cv_landing_view.dart:99]: Better formatting can be done here.

File: lib/ui/views/startup_view.dart [hunk 2]
```diff
@@ -9,14 +8,11 @@ class StartUpView extends StatelessWidget {
...
-          child: Text(
-            'CircuitVerse',
-            style: Theme.of(context).textTheme.headline4.copyWith(
-                  color: Colors.white,
-                ),
-          ),
+          child: Container(
+              padding: const EdgeInsets.symmetric(horizontal: 42),
+              child: Image.asset('assets/images/landing/cv_full_logo.png',
+                  key: Key('cv_startup_logo'))),
```
Comment (Nitish145) [inline lib/ui/views/startup_view.dart:15]: Better formatting

File: lib/ui/views/cv_landing_view.dart (hunk 1 above)
Comment (manjotsidhu) [reply lib/ui/views/cv_landing_view.dart:99]: Sure

File: lib/ui/views/startup_view.dart (hunk 2 above)
Comment (manjotsidhu) [reply lib/ui/views/startup_view.dart:15]: 👍

File: lib/ui/views/cv_landing_view.dart (hunk 1 above)
Comment (manjotsidhu) [reply lib/ui/views/cv_landing_view.dart:99]: The code is formatted already using `flutter format`

Comment (manjotsidhu) [reply lib/ui/views/cv_landing_view.dart]: Done!

File: lib/ui/views/startup_view.dart (hunk 2 above)
Comment (manjotsidhu) [reply lib/ui/views/startup_view.dart]: Done, please check

**Comment by manjotsidhu** [comment]
@Nitish145 @nitin10s the reviewed changes have been made on the pull request.

**Review by tachyons — APPROVED** [review]

File: lib/ui/views/startup_view.dart [hunk 3]
```diff
@@ -9,14 +8,13 @@ class StartUpView extends StatelessWidget {
...
-            style: Theme.of(context).textTheme.headline4.copyWith(
-                  color: Colors.white,
-                ),
-          ),
+          child: Container(
+              padding: const EdgeInsets.symmetric(horizontal: 42),
+              child: Image.asset(
+                'assets/images/landing/cv_full_logo.png',
+                key: Key('cv_startup_logo'),
+              )),
```
Comment (Nitish145) [inline lib/ui/views/startup_view.dart:17]: Better formatting!
One thing is try to use commas wherever possible as in to help `flutter format` format the code.

**Review by Nitish145 — APPROVED** [review]

File: lib/ui/views/startup_view.dart (hunk 3 above)
Comment (manjotsidhu) [reply lib/ui/views/startup_view.dart]: Yep and I have added comma there as you requested.

Comment (Nitish145) [reply lib/ui/views/startup_view.dart]: It's better to not have consecutive brackets.

Comment (manjotsidhu) [reply lib/ui/views/startup_view.dart]: Yes, you can check the updated code.

Comment (Nitish145) [reply lib/ui/views/startup_view.dart]: Code consistency is the key, thanks..

Comment (manjotsidhu) [reply lib/ui/views/startup_view.dart]: Definitely, 👍

**Pull request merged**